testgrid/config/config.yaml

```diff
@@ -3340,6 +3348,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-7
   - name: gce-kubeadm-1.6-on-1.7
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7
+  - name: gce-kubeadm-upgrade-1.6-1.7
+    test_group_name: periodic-ci-kubernetes-e2e-kubeadm-gce-upgrade-1-6-1-7
+  - name: gce-kubeadm-upgrade-1.7-1.8
+    test_group_name: periodic-ci-kubernetes-e2e-kubeadm-gce-upgrade-1-7-1-8
```

Comment:
nit: 1.7-1.8 upgrade job probably should be in release-1.8 dashboard

Reply:
let's take that a bit later